arm/template.json

```diff
@@ -0,0 +1,192 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "appPackageUrl": {
+            "defaultValue": "https://github.com/microsoft/Service-Fabric-POA/releases/download/v1.4.5/PatchOrchestrationApplication_v1.4.5.sfpkg",
```

Comment:
Can you change this to the current latest release of POA v1.4.7 ? For every subsequent release, we will keep on updating the version in the template too.